app/controllers/models_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ModelsController < ApplicationController
+  before_action :set_model, except: :create
```

Comment:
sửa dụng `except` sẽ có nguy cơ rủi ro hơn so với `only`.
Ví dụ sau này một ai đó cần define method new, show....?1?

Reply:
Ok I understand the issue